pandas/core/arrays/arrow/accessors.py

```diff
@@ -13,13 +20,194 @@
     from pandas.core.dtypes.dtypes import ArrowDtype
 
 if TYPE_CHECKING:
+    from collections.abc import Iterator
+
     from pandas import (
         DataFrame,
         Series,
     )
 
 
-class StructAccessor:
+class _ArrowAccessor(metaclass=ABCMeta):
```

Comment:
```suggestion
class ArrowAccessor(metaclass=ABCMeta):
```

Nit: No need to make this private

Reply:
Done, thanks,